demo/csi/hostpath/redis.nomad

```diff
@@ -42,4 +42,4 @@ job "example" {
       }
     }
   }
-}
+}
```

Comment:
Little bit of debris to cleanup here.

Reply:
I cant remove it :s

Reply:
You can remove it if you flatten the commits. Or you can update your editor config so that you make sure it's always adding the newline EOF (you want that for posix-compliance... run `wc -l` on this file and you'll see it's one line short!)